src/components/Resources.js

```diff
@@ -27,6 +27,12 @@ const links = [
     length: "1 min read",
     url: "docs/keploy-explained/why-keploy",
   },
+  {
```

Comment:
@Sonichigo Like we discussed it'll be good to have a separate section instead of hacktoberfest just as a quick link.

Reply:
@nehagup Done! Please check #93